### PR TITLE
Add operation get-deliverables-analysis command

### PR DESCRIPTION
**Pull request by janinko**
### Checklist:

* [ ] Have you added a note in the [CHANGELOG wiki](https://github.com/project-ncl/bacon/wiki/Changelog) for your change if user-facing?
* [ ] Have you added unit tests for your change?

### CODE REVIEW

File: pnc/src/main/java/org/jboss/pnc/bacon/pnc/OperationCli.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+import org.jboss.pnc.bacon.common.cli.AbstractGetSpecificCommand;
+import org.jboss.pnc.bacon.pnc.common.ClientCreator;
+import org.jboss.pnc.client.ClientException;
+import org.jboss.pnc.client.OperationClient;
+import org.jboss.pnc.dto.DeliverableAnalyzerOperation;
+import picocli.CommandLine.Command;
+
+@Slf4j
+@Command(name = "operation", description = "Long running operations", subcommands = { OperationCli.Get.class })
+public class OperationCli {
```
Comment (jomrazek) [inline pnc/src/main/java/org/jboss/pnc/bacon/pnc/OperationCli.java:30]: I don't know, looks like part of deliverable analysis, either we can create separate command group for all deliverable commands or put this into product milestone where rest of them lives. @thescouser89 ?

**Review by jomrazek — CHANGES_REQUESTED** [review]

File: pnc/src/main/java/org/jboss/pnc/bacon/pnc/OperationCli.java (hunk 1 above)
Comment (janinko) [reply pnc/src/main/java/org/jboss/pnc/bacon/pnc/OperationCli.java:30]: there will be other operations in the future, but I'm fine if you find some nice place for it

Comment (jomrazek) [reply pnc/src/main/java/org/jboss/pnc/bacon/pnc/OperationCli.java:30]: you mean other than ProductMilestoneCli where rest of them are?

Comment (thescouser89) [reply pnc/src/main/java/org/jboss/pnc/bacon/pnc/OperationCli.java:30]: yeah @jomrazek , a separate group for deliverables would be nice. Same as for DA.

**Review by jomrazek — APPROVED** [review]

**Pull request merged**